examples/usgs-3dep-1/collection.json

```diff
@@ -0,0 +1,64 @@
+{
+  "type": "Collection",
```

Comment:
General comment - we should strive to use the item-assets extension to define all possible assets for the Collection.  It's really helpful to be able to look at the Collection and see what asset keys, descriptions etc. are available

Reply:
I know this is probably in one of the STAC "best practices" doc, but what do you reckon is the best way to enforce this on STAC content creators? Putting an `INFO`-type message into `stac validate` type commands? Hope they Read The Damn Manual™? Make them pass an empty list for `stac_extensions` to prove they're really serious about not using `proj` and `item_assets`?